README.md

```diff
@@ -102,6 +104,11 @@ OPTIONS:
                                               [possible values: http, ipc]
 ```
 
+## RPC Methods
+- `discv5_nodeInfo`     Returns the ENR of the client
+- `web3_clientVersion`  Returns the current version of Trin being run
+- `eth_blockNumber `    Returns the current block number from the tip fo the chain (as provided by Infura at present)
```

Comment:
Hm, this is going to turn into a very long list, if it becomes comprehensive. It's maybe fine to just have a short selection of these, but maybe it's worth also linking to the json-rpc wiki, and only listing things that *aren't* in the wiki.

```suggestion
## Custom RPC Methods
- `discv5_nodeInfo`     Returns the ENR of the client
- `discv5_routingTableInfo`     Returns the list of discovery peers that have recently been available

See https://eth.wiki/json-rpc/API#json-rpc-methods for other standard methods that are implemented. Most of the currently proxy to Infura.
```